CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Feature
 
+- Added new Block Style Wrapper. This implementation is marked as **experimental** during Volto 16 alpha period. The components, API and the styling are subject to change **without issuing a breaking change**. You can start using it in your projects and add-ons, but taking this into account. See documentation for more information. @sneridagh
```

Comment:
Is this still experimental, now that it is a non-breaking change?

Reply:
This is an unmentioned fact, every feature added in the alpha period IS in fact experimental. I initially thought that it might be interesting to really note it and being explicit in this case, instead of being implicit... But it's true that kind of lead to confusion now...